app/models/concerns/hostext/search.rb

```diff
@@ -92,6 +93,7 @@ module Search
         scoped_search :relation => :search_users, :on => :firstname, :complete_value => true, :only_explicit => true, :rename => :'user.firstname',:operators => ['= ', '~ '], :ext_method => :search_by_user
         scoped_search :relation => :search_users, :on => :lastname,  :complete_value => true, :only_explicit => true, :rename => :'user.lastname', :operators => ['= ', '~ '], :ext_method => :search_by_user
         scoped_search :relation => :search_users, :on => :mail,      :complete_value => true, :only_explicit => true, :rename => :'user.mail',     :operators => ['= ', '~ '], :ext_method => :search_by_user
+        scoped_search :relation => :usergroups,   :on => :name,      :complete_value => true, :only_explicit => true, :rename => :usergroup
```

Comment:
For consistency, renaming to usergroup.name would be better I think

Reply:
Is it necessary if we cannot search by other attributes ?

Reply:
perhaps not, it might be just confusing why for user you'd use user.login (few lines above), perhaps use `, :aliases => [:usergroup]`